tests/test_never_type/test_runtime.py

```diff
@@ -0,0 +1,28 @@
+from pytest import mark, raises
+
+from basedtyping import Never, issubform
+
+# xfail tests due to https://github.com/KotlinIsland/basedtyping/issues/22
+
+
+@mark.xfail
+def test_isinstance() -> None:
+    assert not isinstance(
+        1,
+        # https://github.com/KotlinIsland/basedmypy/issues/136
+        Never,  # type:ignore[arg-type]
+    )
+
+
+def test_issubform_true() -> None:
+    assert issubform(Never, int)
+
+
+@mark.xfail
```

Comment:
Shouldn't this pass now?

Reply:
no because i removed th `__subclasscheck__`

Reply:
`test_issubform_true` is the one that now passes

Reply:
`assert not issubform(str, Never)`